share/spack/spackos-bootstrap/stage1/spack.in.yaml

```diff
@@ -0,0 +1,19 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  include:
+    - compilers.yaml
+  specs:
+  - binutils +stage1
+  - gcc @13.2.0 +stage1 +binutils ~bootstrap languages=c,c++
```

Comment:
Not related to this PR, but we have:
```
"LDFLAGS=-static-libstdc++ -static-libgcc "
```
set by `gcc` recipe, which we may want to get rid of?

Reply:
It seems these flags were set back in the days in f8e046bb33b23890e1e3bf92866bd6ea5cefbbec

Reply:
Wow. That's from a while ago.  It's useful to have those for early stages, though we don't necessarily need them for later ones.